src/brasil/gov/tiles/tiles/navigation.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+from Acquisition import aq_parent
+from brasil.gov.tiles import _
+from collective.cover.tiles.base import IPersistentCoverTile
+from collective.cover.tiles.base import PersistentCoverTile
+from plone import api
+from plone.dexterity.interfaces import IDexterityContent
+from Products.Five.browser.pagetemplatefile import ViewPageTemplateFile
+from zope.interface import implementer
+
+
+class INavigationTile(IPersistentCoverTile):
+    """Display a navigation of items."""
+
+
+@implementer(INavigationTile)
+class NavigationTile(PersistentCoverTile):
+    """Display a navigation of items."""
+
+    index = ViewPageTemplateFile('templates/navigation.pt')
+    short_name = _(u'msg_short_name_navigation', default=u'Navigation')
+
+    is_configurable = False
+    is_droppable = False
+    is_editable = False
+
+    def __init__(self, context, request):
+        super(NavigationTile, self).__init__(context, request)
+        self._setup()
+
+    def _setup(self):
+        self.menu_items = []
+        self.session = aq_parent(self.context)
+        try:
+            for o in self.context.listFolderContents():
```

Comment:
IIRC, you can filter stuff in `listFolderContents()`, check if you can set the workflow state on it.

Reply:
refactoring for `api.content.find`